### DISCUSSION TITLE
Not able to get Github education. HELP

### DISCUSSION THREAD

**Discussion by TheAltintas** [community/community#139160] [GitHub Education]
### Hi everyone,

Hej Everyone 

Recentely i tried to apply for github education with everything thats needed, yet I can't seem to apply.

It doens't allow me to. My classmates from school were able to do it but not me?

Under here is the link showing that it didnt allow it the first time. 

But now I can't seem to apply again.

![xGithub](https://github.com/user-attachments/assets/2d98ab11-7d04-4726-ac3b-076bfd7aef96)

**Comment by alx-mp4** [comment]
Hi @TheAltintas, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

**Reply by TheAltintas** [reply]
How do i open a ticket with Github support?

I have no idea

**Reply by alx-mp4** [reply]
Here you go https://support.github.com/contact
